recipe/build.sh

```diff
@@ -11,7 +11,7 @@ then
     export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11"
 fi
 
-./configure --prefix="${PREFIX}"
+./configure --prefix="${PREFIX}" --enable-unicode
```

Comment:
Weird, the readme says it is --enable-unicode: http://www.pcre.org/readme.txt  Has there been a version bump?